lib/internal/Magento/Framework/Bulk/BulkStatusInterface.php

```diff
@@ -38,9 +38,20 @@ public function getOperationsCountByBulkIdAndStatus($bulkUuid, $status);
      * @param int $userId
      * @return BulkSummaryInterface[]
      * @since 100.2.0
+     * @deprecated 100.3.0
      */
     public function getBulksByUser($userId);
 
+    /**
+     * Get all bulks created by user and user type
+     *
+     * @param int $userId
+     * @param int $userTypeId
+     * @return BulkSummaryInterface[]
+     * @since 100.2.0
+     */
+    public function getBulksByUserAndType($userId, $userTypeId);
```

Comment:
Adding the new methods to the interface is backward-incompatible change. Please, introduce new interface with this method.